lib/dfe/reference_data/itt/cycles.rb

```diff
@@ -101,6 +101,18 @@ def self.make_local_time(*args)
               christmas: Date.new(2023, 12, 18)..Date.new(2024, 1, 5),
               easter: Date.new(2024, 3, 18)..Date.new(2024, 4, 1)
             }
+          },
+          '2024-2025' => {
+            find_opens: make_local_time(2024, 10, 1, 9), # First Tuesday of October
+            apply_opens: make_local_time(2024, 10, 8, 9), # Second Tuesday of October
+            # apply_1_deadline: make_local_time(0, 0, 0, 0), # 1st Tuesday of September
+            apply_2_deadline: make_local_time(2025, 9, 15, 18), # 2 weeks after Apply 1 deadline
```

Comment:
I believe this should be the 3rd Tuesday in September so 16th September

Reply:
👍 now updated